api/openapi_server/models/database.py

```diff
@@ -40,6 +40,14 @@ class Applicant(Base):
     applicant_status = Column(Integer, ForeignKey('applicant_status.id'), nullable=False)
     user = Column(Integer, ForeignKey('user.id'), nullable=False)
 
+class GuestApplications(Base):
+    __tablename__ = "guest_applications"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
I think specifying primary_key is enough to create an index. `index` might be redundant here. not sure but just noting for future reviewers